app/clients/youtube/video_statistics.rb

```diff
@@ -0,0 +1,24 @@
+module Youtube
+  class VideoStatistics < Client
+    def list(video_id)
+      path = "/videos"
+      query = {
+        part: "statistics",
+        id: video_id
+      }
+
+      response = all_items(path, query: query)
+
+      if response.empty?
+        puts "Error: No video found with the given ID"
+        return nil
+      end
+
+      # Assuming there's only one item in the response, as the ID of a video is unique.
+      {
+        view_count: response.first["statistics"]["viewCount"],
+        like_count: response.first["statistics"]["likeCount"]
+      }
```

Comment:
What are the other values returned by the API. Should we return them from that method in case we need them in the future?

Reply:
It has comment and favorite count additionally. Do we need those? (Not sure what is a favorite, probably something outdated)